libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/FeedbackEvent.kt

```diff
@@ -7,105 +7,105 @@ import androidx.annotation.StringDef
-         */
-        const val FEEDBACK_SOURCE_UI = "user"
-    }
+object FeedbackEvent {
+    /**
+     * Feedback type *general*: when no one else is not fit or actually *general* feedback
+     */
+    const val GENERAL_ISSUE = "general"
+
+    /**
+     * Feedback type *accident*: crash, uneven surface, etc
+     */
+    const val ACCIDENT = "accident"
+
+    /**
+     * Feedback type *hazard*: such as debris, stopped vehicles, etc
+     */
+    const val HAZARD = "hazard"
+
+    /**
+     * Feedback type *road closed*: closed road or one that does not allow vehicles
+     */
+    const val ROAD_CLOSED = "road_closed"
+
+    /**
+     * Feedback type *not allowed*: a turn/maneuver that isn't allowed
+     */
+    const val NOT_ALLOWED = "not_allowed"
+
+    /**
+     * Feedback type *routing error*: poor instruction or route choice
+     * (ambiguous or poorly-timed turn announcement, or a set of confusing turns)
+     */
+    const val ROUTING_ERROR = "routing_error"
+
+    /**
+     * Feedback type *missing road*: road doesn't exist
+     */
+    const val MISSING_ROAD = "missing_road"
+
+    /**
+     * Feedback type *missing exit*: dead end
+     */
+    const val MISSING_EXIT = "missing_exit"
+
+    /**
+     * Feedback type *confusing instruction*: wrong voice/text instruction
+     */
+    const val CONFUSING_INSTRUCTION = "confusing_instruction"
+
+    /**
+     * Feedback type *inaccurate gps*: wrong location of navigation puck on the map
+     */
+    const val INACCURATE_GPS = "inaccurate_gps"
+
+    /**
+     * Feedback type *incorrect visual guidance*: wrong visual guidance
+     */
+    const val INCORRECT_VISUAL_GUIDANCE = "incorrect_visual_guidance"
+
+    /**
+     * Feedback type *incorrect audio guidance*: wrong audio guidance
+     */
+    const val INCORRECT_AUDIO_GUIDANCE = "incorrect_audio_guidance"
+
+    /**
+     * Feedback source *reroute*: the user tapped a feedback button in response to a reroute
+     */
+    const val REROUTE = "reroute"
+
+    /**
+     * Feedback source *user*: the user tapped a feedback button
+     */
+    const val UI = "user"
 
     /**
      * Type of feedback mean WHAT happen
      */
     @Retention(AnnotationRetention.SOURCE)
     @StringDef(
-        FEEDBACK_TYPE_GENERAL_ISSUE,
-        FEEDBACK_TYPE_ACCIDENT,
-        FEEDBACK_TYPE_HAZARD,
-        FEEDBACK_TYPE_ROAD_CLOSED,
-        FEEDBACK_TYPE_NOT_ALLOWED,
-        FEEDBACK_TYPE_ROUTING_ERROR,
-        FEEDBACK_TYPE_CONFUSING_INSTRUCTION,
-        FEEDBACK_TYPE_INACCURATE_GPS,
-        FEEDBACK_TYPE_MISSING_ROAD,
-        FEEDBACK_TYPE_MISSING_EXIT
+        GENERAL_ISSUE,
+        ACCIDENT,
+        HAZARD,
+        ROAD_CLOSED,
+        NOT_ALLOWED,
+        ROUTING_ERROR,
+        MISSING_ROAD,
+        MISSING_EXIT,
+        CONFUSING_INSTRUCTION,
+        INACCURATE_GPS,
+        INCORRECT_VISUAL_GUIDANCE,
+        INCORRECT_AUDIO_GUIDANCE
     )
-    annotation class FeedbackType
+    annotation class Type
```

Comment:
are you sure you want to change this to Type? to be consistent, looks you'll want to follow up and change all of these 🤔 

![Screen Shot 2020-04-28 at 9 44 41 AM](https://user-images.githubusercontent.com/3021882/80514211-0e098b80-8935-11ea-98af-5845f8461548.png)

Reply:
Also there are enough "Type" objects

![Screen Shot 2020-04-28 at 9 53 38 AM](https://user-images.githubusercontent.com/3021882/80515001-24fcad80-8936-11ea-9a72-e9b03df76fc8.png)

Reply:
`BicycleType` is an `enum`. It's also for the _Onboard_ module (not actively under development at the moment) so for now I'm going to leave it as is.
`DirectionsRouteType` is from the legacy.